app/src/sharedTest/java/org/oppia/android/app/topic/revisioncard/RevisionCardFragmentTest.kt

```diff
@@ -110,7 +110,8 @@ class RevisionCardFragmentTest {
     ).use {
       openActionBarOverflowOrOptionsMenu(context)
       onView(withText(context.getString(R.string.menu_options))).check(matches(isDisplayed()))
-      onView(withText(context.getString(R.string.help))).check(matches(isDisplayed()))
+      onView(withText(context.getString(R.string.help_activity_title)))
```

Comment:
Use `help_menu` here as it is part of menu.

Reply:
changed help_activity_title to menu_help